frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterDetails.tsx

```diff
@@ -154,6 +158,12 @@ export default function ClusterDetailsPage({ match }: RouteComponentProps<{ id:
 
     const clusterExists = !!managedCluster || !!clusterDeployment || !!managedClusterInfo || !!hostedCluster
 
+    const clusters = useAllClusters()
+    const selectedHostedCluster = clusters.find((c) => c.name === match.params.id)
```

Comment:
For this, I'm wondering can you re-use the hostedCluster from https://github.com/stolostron/console/blob/3fc25fd0eb7264e7286711352fe8940bde18240b/frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterDetails.tsx#L146 so we don't have to find it twice?

Reply:
kk thanks @fxiang1 , will try that